cli/flox/src/commands/containerize/macos_containerize_proxy.rs

```diff
@@ -1,15 +1,19 @@
 use std::convert::Infallible;
+use std::env;
 use std::path::PathBuf;
 
 use dirs::home_dir;
-use flox_rust_sdk::flox::FLOX_VERSION;
+use flox_rust_sdk::flox::{Flox, FLOX_VERSION};
 use flox_rust_sdk::providers::container_builder::{ContainerBuilder, ContainerSource};
+use once_cell::sync::Lazy;
 
 use super::Runtime;
 use crate::config::FLOX_DISABLE_METRICS_VAR;
 
 const FLOX_FLAKE: &str = "github:flox/flox";
 const FLOX_PROXY_IMAGE: &str = "ghcr.io/flox/flox";
+pub static FLOX_CONTAINERIZE_FLAKE_REF_OR_REV: Lazy<Option<String>> =
+    Lazy::new(|| env::var("FLOX_CONTAINERIZE_FLAKE_REF_OR_REV").ok());
```

Comment:
suggestion: use `std::sync::LazyLock` which is the std replacement for the `once_cell` types.
That has only become available in the last year or so, so not all or uses of `Lazy` have been replaced yet, but for new uses we should probably move to using the `std`.

Luckily the interface is the same.

Reply:
Fixed